scripts/autoscaling/init_multicloud_v1.15.4.sh

```diff
@@ -211,6 +211,12 @@ if [ $? -ne 0 ]; then
   _DOCKER_SYS_IMGS="/opt/docker-system-images"
 fi
 
+
+cd /bin
+git clone https://github.com/magnific0/wondershaper.git
+cd wondershaper
+sudo make install
```

Comment:
rewrite to 
```
cd /bin && \
git clone https://github.com/magnific0/wondershaper.git && \
cd wondershaper && \
sudo make install
```

at the end also check if installation was successful and print log about the result